ansible_collections/arista/avd/roles/eos_cli_config_gen/templates/eos/event-monitor.j2

```diff
@@ -1,6 +1,7 @@
-{% if event_monitor is defined and event_monitor is not none %}
-{%     if event_monitor.enabled == true %}
+{# eos - Event monitor #}
+{% if event_monitor is arista.avd.defined %}
+{%     if event_monitor.enabled is arista.avd.defined(true) %}
```

Comment:
Both if statement can be merged as `arista.avd.defined` does a deep test

Reply:
line 2 is redundant and should be removed